src/pages/blog/the-perfect-react-bootcamp-student.md

```diff
@@ -0,0 +1,92 @@
+
+- Modern JS
+
+- Forms
+
+- Authentication
+
+- Styling
+
+- Redux
+
+- Testing principles and testing in React
+
+- Functional programming
+
+- Advanced React patterns
+
+- GraphQL
+
+Obviously, this kind of React training and GraphQL course can be pretty intense. It isn’t for the faint-hearted, but we make sure classes are small enough so that students get the most out of their face-to-face React and GraphQL training.
+
+**To get the most out of the Bootcamp, you’ve got to be at the right stage in your career and at a certain level of experience.**
+
+![image alt text](https://firebasestorage.googleapis.com/v0/b/reactjsacademy-react.appspot.com/o/blog%20post%20images%2FperfectReactStudent%2Fimage_2.jpg?alt=media&)
+
+## How much experience do I need to learn React?
+
+As I said above, the React Bootcamp is intense and rapid. **We’re not here to introduce you to coding, we’re here to supplement your knowledge and introduce you to a new technology.** So the students who get the most out of our React training course are experienced developers.
+
+If you’re familiar with JavaScript but want to learn a new, efficient way to use it - and fast! - you’re ready for the React Bootcamp.
+
+Although the course is intense, it’s worth it. One former student, Polina Stoyanova from tray.io, said she found the course "fantastic".
+
+> "It was really tiring, but it was all worth it. [...] Overall, it taught me many things I didn’t know that I didn’t know about React! So I would definitely recommend putting the time and effort in."
+
+<video youtube-id="6hmKu1-vW-8" ></video>
+
+On another note, if you’re a manager the Bootcamp is perfect for levelling out your team. Our course will get your devs up to speed and enhance the knowledge of your best employees. Everyone will be on the same wavelength afterwards, and that team cohesion will pay dividends for you going forward. Find out more about how team training can level your team’s skills [on our blog](https://reactjs.academy/blog/5-reasons-why-advanced-dev-training-will-keep-your-company-tech-relevant/).
+
+## Who will get the most out of the React Bootcamp?
+
+But it isn’t just about experience. **The ideal RJSA student is passionate about their work and loves what they do.** You’ll get the most out of our week-long training course if you engage with the material and are able to pick up new ideas quickly.
+
+Similarly, the React training is great for helping you or your team progress in their career. You’ll diversify your skillset to futureproof your career.
+
+That’s the opinion of Senior Software Developer, Davian Robinson anyway. After one of our London React Bootcamps, he said:
+
+> "Don’t be afraid - start right now. Book as quickly as possible because this is a very important skillset in the industry. That’s what I’d recommend - just go for it!"
+
+<video youtube-id="8C09x9D_4kk" ></video>
+
+##Why should I learn React? Is it easy to adopt?
```

Comment:
@ric9176 @shakersinthedark I found out that adding an anchor in the titles helps for link building. Example, you can have at the bottom a "wrap up / recap" that includes a link to those titles. Then when you run the auto import of the post in Medium we get a lot of link to our website. Example, the conclusion at the bottom (https://reactjs.academy/blog/react-forms-controlled-and-uncontrolled-components/) has a link to the section "Form library" https://reactjs.academy/blog/react-forms-controlled-and-uncontrolled-components/#form-library. And it's also the same in Medium https://medium.com/leanjs/react-forms-controlled-uncontrolled-components-7842a7284dfa, though the Medium links take the user to the section in our blog

Reply:
Yeah good shout - I'll add that asap